sonar-project.properties

```diff
@@ -0,0 +1,11 @@
+sonar.organization=wednesday-solutions
+sonar.projectKey=wednesday-solutions_next-bulletproof-ts_AY6yu6eKB2n8RRmGoUz4
+
+sonar.language=ts
+sonar.sources=.
+sonar.tests=app
+sonar.exclusions=**/tests/**/*.*,**/storybook/**/*.*,**/internals/**/*.*,**/server/**/*.*,app/**/*.test.{ts,tsx},app/*/RbGenerated*/*.{ts,tsx},app/app.ts,app/global-styles.ts,app/*/*/loadable.{ts,tsx},**/loadable.ts,**/apiUtils.ts,**/testUtils.ts,**/stories/**,**/themes/index.ts
```

Comment:
are all these still applicable for this project?

Reply:
will remove it for now

Reply:
let's note remove it, lets figure what is applicable and add only those